doc/sphinx-guides/source/installation/installation-main.rst

```diff
@@ -157,49 +157,30 @@ If your mail host requires a username/password for access, continue to the next
 Mail Host Configuration & Authentication
```

Comment:
Do we also need to re-write the section above called "Problems Sending Email"? (I think so.) Or delete it?

Reply:
IMHO we should delete it. Any problems with a non-existing hostname etc will appear in the logs as soon as mails are being sent.

Reply:
Ok. In ac74b23a4e I deleted it. Thanks.